internal/pkg/apply/apply.go

```diff
@@ -43,89 +48,294 @@ type Apply struct {
 
 	// Commit is a git commit object
 	Commit *object.Commit
+
+	// Flag; deletes objects previously applied, but not applied currently.
+	Prune bool
+
+	// Flag; don't actually persist created/updated/deleted resources.
+	DryRun bool
+
+	// List of previous inventory objects associated with director.
+	PrevInventory []unstructured.Unstructured
```

Comment:
Suggest change []unstructured.Unstructured to []*unstructured.Unstructured since the sources are defined as that type.